web/themes/joinup/prototype/sass/components/_fieldset.sass

```diff
@@ -71,6 +79,18 @@
       right: -60px
       left: -60px
 
+.fieldset--textarea
+  p
+    overflow: hidden
```

Comment:
Seems really risky to do that.

Reply:
Implementation of the `fieldset--textarea` and `fieldset--border-bottom` have been slightly changed to avoid adding any styles on the paragraph.